src/sentry/workflow_engine/migrations/0051_migrate_remaining_issue_alerts.py

```diff
@@ -30,8 +30,8 @@
     "sentry.rules.conditions.first_seen_event.FirstSeenEventCondition": "condition/event",
     "sentry.rules.conditions.regression_event.RegressionEventCondition": "condition/event",
     "sentry.rules.conditions.reappeared_event.ReappearedEventCondition": "condition/event",
-    "sentry.rules.conditions.new_high_priority_issue.NewHighPriorityIssueCondition": "condition/event",
-    "sentry.rules.conditions.existing_high_priority_issue.ExistingHighPriorityIssueCondition": "condition/event",
+    "sentry.rules.conditions.high_priority_issue.NewHighPriorityIssueCondition": "condition/event",
+    "sentry.rules.conditions.high_priority_issue.ExistingHighPriorityIssueCondition": "condition/event",
```

Comment:
hmm - i feel like we shouldn't modify the migrations that have already run - doesn't that break the idempotency of the migration?

Reply:
yeah i can change back